components/list/list.js

```diff
@@ -4,7 +4,9 @@ class List extends LitElement {
 	static get properties() {
 		return {
 			dividerMode: { type: String, attribute: 'divider-mode' },
-			dividerExtend: {type: Boolean, attribute: 'divider-extend'}
+			dividerExtend: { type: Boolean, attribute: 'divider-extend'},
+			hoverEffect: {type: Boolean, attribute: 'hover-effect'},
+			selectable: {type: Boolean }
```

Comment:
I think we are making an assumption (with `selectable:Boolean`) that selection is always multiple/checkboxes.  The MVC list supports single selection, which I think was used very rarely.  This _might_ be ok, but it's worth calling out, given that it has consequence for the public API and changing later could be painful.

Reply:
This is interesting. I did make the assumption that if the list has selectable element that all elements are selectable. I'll have to talk to Rebecca and Colin about it.

I did make radio selection not part my work but that will be added in the future. Do you think I should make it: `selectable: [checkbox|radio|none(default)]`